parts/linux/cloud-init/artifacts/cse_cmd.sh

```diff
@@ -150,6 +150,7 @@ KUBENET_TEMPLATE="{{GetKubenetTemplate}}"
 CONTAINERD_CONFIG_CONTENT="{{GetContainerdConfigContent}}"
 CONTAINERD_CONFIG_NO_GPU_CONTENT="{{GetContainerdConfigNoGPUContent}}"
 IS_KATA="{{IsKata}}"
+ARTIFACT_STREAMING_ENABLED="{{ArtifactStreamingEnabled}}"
```

Comment:
if we follow the other fields, this should have a prefix like "Is", "Has" or "Get".
For ArtifactStreaming, maybe "IsArtifactStreamingEnabled" is appropriate?

Reply:
There is a mix of fields, like `FIPSEnabled` and others which have such a prefix. I think either one is fine for NBC. What do you think?

Reply:
I personally like `ENABLE_ARTIFACT_STREAMING` a bit better, along with `EnableArtifactStreaming`, which is what you already have :)

Reply:
Discussed and we want to keep the state, so switched to a combination of both.